settings.gradle.kts

```diff
@@ -0,0 +1,24 @@
+rootProject.name = "text"
+includeBuild("build-logic")
+include(":sonar-text-plugin")
+if (System.getenv("SONARSOURCE_QA") != null || providers.gradleProperty("enable_its").isPresent) {
```

Comment:
This solution has one big drawback: The ITs are not included in the local build, so even compilation is not executed. I prefer the way it is solved in sonar-kotlin, where only `test` execution is disabled. 
My dream solution is to have a custom `integrationTests` task in plugin/ruling subprojects and an empty `test` task. But this can be done as a separate ticket.

Reply:
Yes, for now I added the property `enable_its=true` to my `gradle.properties` to have them locally, but indeed we should think of a better solution

Reply:
Is there a ticket created for it?

Reply:
https://sonarsource.atlassian.net/browse/SONARTEXT-96